# This msg file specifies the object pose which includes 
# (1) the name of the object
# (2) the pose of the object (position and orientation)

string object_name
geometry_msgs/Pose object_pose